projects/models/project.py

```diff
@@ -58,15 +55,37 @@ def __str__(self):
         return self.name
 
 
-class ProjectPhaseAttribute(models.Model):
+class ProjectPhaseSection(models.Model):
+    phase = models.ForeignKey(ProjectPhase, verbose_name=_('phase'), related_name='sections', on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, verbose_name=_('name'))
+    index = models.PositiveIntegerField(verbose_name=_('index'), null=True, blank=True)
```

Comment:
Considering how the indexes are used now, I don't think there is any reason to make them nullable(?)